### PR TITLE
[UI] Add WebKitGTK as a product

### PR DESCRIPTION
**Pull request by stephenmcgruer**
This adds WebKitGTK as a product to the UI. This should not affect API endpoints.

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
(This is WIP)

**Comment by stephenmcgruer** [comment]
@Hexcles PTAL. I couldn't think of much in the way of testing for this change, but (very) happy to do any if you can think of something worthwhile. (Maybe just a dumb test that setting `products` via templating does override the rendered result properly?)

File: webapp/components/product-info.js [hunk 1]
```diff
@@ -97,6 +109,13 @@ function parseProduct(name) {
 const ProductInfo = (superClass) => class extends superClass {
   static get properties() {
     return {
+      // Polymer templates can only access variables in the scope of the owning
+      // class. Forward some declarations so that subclasses can use them in
+      // template parameters.
+      allProducts: {
+        type: Array,
+        value: AllProducts,
```
Comment (Hexcles) [inline webapp/components/product-info.js]: Nit: mark this as `readOnly: true` to prevent accidental modification.

Comment (stephenmcgruer) [reply webapp/components/product-info.js]: Nice, didn't know that was an available property. Done!

**Pull request merged**